airbyte-integrations/connectors/destination-bigquery/metadata.yaml

```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: database
   connectorType: destination
   definitionId: 22f6c74f-5699-40ff-833c-4a879ea40133
-  dockerImageTag: 1.5.7
+  dockerImageTag: 1.6.0
```

Comment:
minor bump since it's a breaking change for a beta feature...? idk, semver gets a little weird in this case